stable/influxdb/values.yaml

```diff
@@ -61,6 +61,48 @@ setDefaultUser:
     ## Default: "WITH ALL PRIVILEGES"
     # privileges: "WITH ALL PRIVILEGES"
 
+## Create database through Kubernetes job
+## Defaults indicated below
+##
+createDatabase:
+  enabled: false
+
+  ## Image of the container used for job
+  ## Default: appropriate/curl:latest
+  ##
+  #image: appropriate/curl:latest
+
+  ## Deadline for job so it does not retry forever.
+  ## Default: activeDeadline: 300
+  ##
+  #activeDeadline: 300
+
+  ## Restart policy for job
+  ## Default: OnFailure
+  #restartPolicy: OnFailure
+
+  #database:
+
+    ## The user name
+    ## Default: "influxdb"
```

Comment:
If there is any default on this chart I would like it to be `prometheus`. There have been some recent changes to kapacitor that allow it to auto-discover and scrape `/metrics` endpoints. That data goes into the `prometheus` database.